parametric/apps/nodejs/servicer.js

```diff
@@ -26,11 +26,11 @@ class Servicer {
             parent.origin = request.origin;
         }
 
-        const { http_headers = {} } = request.http_headers || {};
+        const http_headers = request.http_headers.http_headers || [];
```

Comment:
IIRC there were case where the first level of http_headers would be undefined which was why it was done with the destructuring previously. Might need to check if this still works. 🤔

Reply:
With the existing code, I tested a case where I passed `None` into one of the python methods that expected an array, and it seems to work okay 🤷🏼 Maybe we can leave as-is for now?